### PR TITLE
can't get mongoose working

### PR DESCRIPTION
**Pull request by carolyngreeno**
I typed ‘mongo’ in one terminal and ‘mongo’ in another and it seems to
work.

Then I tried this:
‘mongo trucks < models/foodTrucksJSON.js’

And it seemed to work because I got this as part of the message:
"nInserted" : 60,

When I try this, there’s nothing returned. Can you tell me what I might
be doing wrong?
‘db.trucks.find()’

### CODE REVIEW

**Comment by jamdon** [comment]
I'm assuming you're doing this from the mongo shell? If so, then you have to change to using the foodTruckAPI database first. To do that, type: use foodTruckAPI and then try the db.trucks.find() command.

**Comment by carolyngreeno** [comment]
That was the missing step. Thanks!

File: gruntfile.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+		jshint: {
+			all: {
+				src: ['app/**/*.js']
+			}
+		}
+	});
+
+	grunt.loadNpmTasks('grunt-contrib-jshint');
+	//grunt.loadNpmTasks('grunt-mocha');
+	//grunt.registerTask('default', ['jshint', 'mocha']);
```
Comment (jamdon) [inline gruntfile.js:14]: Ideally include a default task. In this case, if you don't want to use the mocha task, just remove it from the list and make jshint the default task.

**Comment by jamdon** [comment]
:+1:

**Pull request closed without merging**